src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -14,27 +14,145 @@
 limitations under the License.
 -->
 
-<div layout="column" layout-margin layout-align="center center">
-  Displaying details for replica set {{ctrl.replicaSetDetail.name}} in namespace
-  {{ctrl.replicaSetDetail.namespace}}.
-  <div>
-    <table>
-      <thead>
-        <tr>
-          <th>Pod</th>
-          <th>Start time</th>
-          <th>IP</th>
-          <th>Node</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr ng-repeat="pod in ctrl.replicaSetDetail.pods">
-          <td>{{pod.name}}</td>
-          <td>{{pod.startTime}}</td>
-          <td>{{pod.podIP}}</td>
-          <td>{{pod.nodeName}}</td>
-        </tr>
-      </tbody>
-    </table>
+<md-content layout="row" layout-fill flex>
+  <div class="kd-replicasetdetail-sidebar" layout="column">
+    <div flex layout-align="start center" layout="row" class="kd-header-option-panel">
+      <a flex-offset="10" ui-sref="replicasets">
+        <i class="material-icons kd-align-mid kd-font-lg">keyboard_backspace</i>
+      </a>
+      <span flex-offset="5" flex class="kd-align-mid md-title">
+        {{ctrl.replicaSetDetail.name}}
+      </span>
+    </div>
+    <div flex-offset="5" flex layout-align="start start" layout-padding layout="row">
+      <div flex layout="column">
+        <h3 layout="row" class="kd-primary md-body-2">
```

Comment:
Hmm... h3 is for content heading. I dont think this is content heading. This is button bar.